commons/src/main/java/org/frankframework/util/XmlEncodingUtils.java

```diff
@@ -119,25 +113,24 @@ public static String decodeChars(String string) {
 	}
 
 	/**
-	   * Conversion of special xml signs. Please note that non valid xml chars
-	   * are not changed, hence you might want to use
-	   * replaceNonValidXmlCharacters() or stripNonValidXmlCharacters() too.
-	   **/
+	 * Conversion of special xml signs. Please note that non-valid xml chars are not changed, hence you might want to use
+	 * {@link #replaceNonValidXmlCharacters(String)} or {@link #stripNonValidXmlCharacters(String, boolean)} too.
+	 **/
 	private static String escapeChar(char c, boolean escapeNewLines) {
 		switch (c) {
-			case ('<') :
+			case ('<'):
```

Comment:
Perhaps you can change to 'enhanced switch' since you're changing it anyway

Reply:
Done!